app/presenters/interaction_presenter.rb

```diff
@@ -11,10 +11,14 @@ def display_name
   end
 
   def na_or_types
-    types = interaction.interaction_types.map{|t| t.type}.reject{|t| t == "n/a"}
+    types = interaction.interaction_types.map{|t| t.type.downcase}.uniq.reject{|t| t == "n/a"}
```

Comment:
This will result in this method returning the downcased version of the type. I assume that is to ensure that the uniq really catches all cases, irregardless of capitalization. However, I would argue that we should instead ensure that the values entered in to the database aren't just differently capitalized duplicates.

Reply:
@susannasiebert Would that involve making sure that importers downcase interactions when they are imported? Or would that be addressed some where else?

Reply:
I think the first thing to decide is if downcasing is what we want or something fancier (e.g. `capitalize` with a special case for `N/A`). I would then suggest to create a function for that conversion in the model for the interaction type which can then be called by all the importers when an interaction type gets created by the importer.